scripts/update/Updater.php

```diff
@@ -45,5 +45,6 @@ public function update($initialVersion)
         }
 
         $this->skip('1.4.0', '1.8.3');
+        $this->skip('1.8.3', '1.9.0');
```

Comment:
no need for 1 more skip(), please use the one above 

        $this->skip('1.4.0', '1.9.0');